lib/sqeduler/trigger_lock.rb

```diff
@@ -15,6 +15,8 @@ def lock
       # - return true if already acquired
       # - refresh the lock if already acquired
       refresh || super
+    rescue RuntimeError
```

Comment:
I used `RuntimeError` because that's the base of all redis errors. But I see your point. Updated.